tests/e2e/tests/devfiles/Python.spec.ts

```diff
@@ -35,15 +40,15 @@ suite(`${workspaceStack} test`, async () => {
     });
 
     suite.skip('Run Python project', async () => {
-        codeExecutionHelper.runTask(taskRunName, 30_000);
-        codeExecutionHelper.closeTerminal(taskRunName);
+        codeExecutionTests.runTask(taskRunName, 30_000);
+        codeExecutionTests.closeTerminal(taskRunName);
     });
 
     suite('Language server validation', async () => {
-        commonLsTests.errorHighlighting(fileName, `error_text;`, 7);
-        commonLsTests.suggestionInvoking(fileName, 9, 22, 'str');
-        commonLsTests.autocomplete(fileName, 9, 4, 'print');
-        // commonLsTests.codeNavigation(tabTitle, 19, 7, codeNavigationClassName); // there is no codenavigation in the Python devfile
+        commonLanguageServerTests.errorHighlighting(fileName, `error_text;`, 7);
+        commonLanguageServerTests.suggestionInvoking(fileName, 9, 22, 'str');
+        commonLanguageServerTests.autocomplete(fileName, 9, 4, 'print');
+        // commonLanguageServerTests.codeNavigation(tabTitle, 19, 7, codeNavigationClassName); // there is no codenavigation in the Python devfile
```

Comment:
If no codenavigation in the Python workspace so maybe remove this commented line?

Reply:
good point, need to verify whether the LS plugin for Python was changed and the feature was added.

Reply:
Will also fix as part of https://github.com/eclipse/che/issues/19749